project/MimaExcludes.scala

```diff
@@ -36,6 +36,9 @@ object MimaExcludes {
 
   // Exclude rules for 2.4.x
   lazy val v24excludes = v23excludes ++ Seq(
+    // [SPARK-6237][NETWORK] Network-layer changes to allow stream upload
+    ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.network.netty.NettyBlockRpcServer.receive"),
```

Comment:
Kinda wondering why this class is public in the first place... along with `SparkTransportConf` in the same package.

Reply:
I suspect that it's because we might want to access these across Java package boundaries and Java doesn't have the equivalent of Scala's nested package scoped `private[package]`.

Reply:
I only see references to them in Scala code... also `private[package]` translates to `public` in Java, so that would at least avoid the mima checks.